### PR TITLE
MOBILE-138: Feed Section Phase 2.1

### PR DESCRIPTION
**Pull request by 07jasjeet**
## Tasks
- [X] Feed Repository and other functions
- [X] Feed Service
- [x] Basic UI implementation

## Surplus
Added auth interceptor for directly injecting access token without unnecessary repeated parameter passing.

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,7 +1,7 @@
 org.gradle.jvmargs=-Xmx4096m
 android.useAndroidX=true
-android.enableJetifier=true
+android.enableJetifier=false
```
Comment (akshaaatt) [inline gradle.properties:3]: I think we should remove this line altogether

**Review by akshaaatt — APPROVED** [review]
Awesome work! 💯

File: gradle.properties (hunk 1 above)
Comment (akshaaatt) [reply gradle.properties:3]: On second though, we should keep it to true as per https://stackoverflow.com/a/56490827/14790459

Comment (07jasjeet) [reply gradle.properties:3]: yeah seems too risky I guess but the fast build times are awesome too. I'm deeply divided xD. Will revert it in the next PR #209.

Comment (akshaaatt) [reply gradle.properties:3]: Cool

**Pull request merged**